helm/templates/ingress.yaml

```diff
@@ -21,6 +16,9 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+{{- if and .Values.ingress.className (semverCompare ">=1.19" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
Just noticed, ingressClassName was introduced in 1.18 so this version should also be included in the condition.

Reply:
I have updated to 1.18 and squashed commits for traceability.